var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -31,6 +31,7 @@ class Papi(AutotoolsPackage, ROCmPackage):
     git = "https://github.com/icl-utk-edu/papi"
 
     version("master", branch="master")
+    version("7.0.1", sha256="c105da5d8fea7b113b0741a943d467a06c98db959ce71bdd9a50b9f03eecc43e")
```

Comment:
Thanks for the PR! Could you please add a comment right below 7.0.1, says 7.0.0 was intentionally omitted due to build issues or something like that.